### PR TITLE
copy(fxa-auth-server): update postConsumeRecoveryCode copy

### PR DESCRIPTION
**Pull request by millsoper**
Because:

* We are updating copy for emails

This commit:

* Updates copy and storybook condition for postConsumeRecoveryCode email

Closes # https://mozilla-hub.atlassian.net/browse/FXA-5197

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Screen Shot 2022-09-09 at 1 24 15 PM

### CODE REVIEW

**Comment by millsoper** [comment]
This is a WIP. The footer will be updated, and tests updated accordingly, when [FXA-5193](https://github.com/mozilla/fxa/pull/14084) is merged and this is rebased on top of it.

**Review by bcolsson — APPROVED** [review]

File: packages/fxa-auth-server/lib/routes/recovery-codes.js [hunk 1]
```diff
@@ -188,6 +188,7 @@ module.exports = (log, db, config, customs, mailer) => {
             uaOSVersion: ua.osVersion,
             uaDeviceType: ua.deviceType,
             uid,
+            numberRemaining: remaining,
```
Comment (chenba) [inline packages/fxa-auth-server/lib/routes/recovery-codes.js:191]: nit: the rest of the keys appear to be in alpha order.

Comment (millsoper) [reply packages/fxa-auth-server/lib/routes/recovery-codes.js]: Good catch, updated!

**Review by chenba — CHANGES_REQUESTED** [review]
The footer doesn't appear to match the one in the doc linked from the jira.  Looks fine otherwise though.

**Comment by millsoper** [comment]
> The footer doesn't appear to match the one in the doc linked from the jira. Looks fine otherwise though.

Do you mean the vertical stacking of the address, or the app badges for downloading the app? I checked with Vesta and she said the app badges are meant to be omitted -- I think I can fix the vertical stacking though!

**Comment by chenba** [comment]
> > The footer doesn't appear to match the one in the doc linked from the jira. Looks fine otherwise though.
> 
> Do you mean the vertical stacking of the address, or the app badges for downloading the app? I checked with Vesta and she said the app badges are meant to be omitted -- I think I can fix the vertical stacking though!

Well I was comparing the verbiage, but then I noticed that there are two footers in that doc and the link in the jira doesn't link to a specific one, so, seems good?

**Review by chenba — APPROVED** [review]

**Pull request merged**